### ISSUE TITLE
Update to MV3

### ISSUE THREAD

**Issue by jacksontriffon** [just-jeb/angular-chrome-extension#12]
Hi @just-jeb. 

The new Manifest-version 3 is becoming mandatory in Jan 2022, but this extension is still on MV2. It would be incredible to see an updated version so that anyone can use Angular within new chrome-extensions.

Personally, I'd love to use this in my next project but I'm finding some challenges migrating to MV3. If you could let me know if this is possible it would inspire me and others to keep using this repo for new endeavours! 

From a fellow Israeli.
SJ

**Comment by just-jeb** [comment]
I completely agree and I'd love to upgrade it to MV3, however it is currently blocked by [this](https://github.com/SimplifyJobs/webpack-ext-reloader/issues/28).
